### PR TITLE
vyos_lldp interfaces model

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

### CODE REVIEW

File: models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      description: A list of lldp interfaces configurations.
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description:
+            - Name of the  lldp interface.
+          type: str
+          required: True
+        disable:
```
Comment (rohitthakur2590) [inline models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml]: change it to enable

File: models/vyos/vyos_lldp_interfaces/deleted_example_01.txt [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Using deleted
+#
+# Before state
+# -------------
+#
+# vyos@vyos# run show configuration commands | grep lldp
+# set service lldp interface eth2 location elin '0000000911'
+#
+- name: Delete lldp  interface ottributes of given interfaces.
```
Comment (samccann) [inline models/vyos/vyos_lldp_interfaces/deleted_example_01.txt]:
```suggestion
- name: Delete LLDP attributes of given interfaces (Note: This won't delete other LLDP configuration).
```

File: models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        'supported_by': 'network'
+    }
+NETWORK_OS: vyos
+RESOURCE: lldp_interfaces
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: vyos_lldp_interfaces
+  version_added: 2.9
+  short_description: Manages attributes of lldp interfaces on VyOS devices.
```
Comment (samccann) [inline models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml]:
```suggestion
  short_description: Manage link layer discovery protocol (LLDP) attributes of interfaces on VyOS devices
```

File: models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  description: This module manages attributes of lldp interfaces on VyOS network devices.
```
Comment (samccann) [inline models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml]:
```suggestion
  description: Manage link layer discovery protocol (LLDP) attributes of interfaces on VyOS network devices.
```

File: models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  author:
+   - Rohit Thakur (@rohitthakur2590)
+  options:
+    config:
+      description: A list of lldp interfaces configurations.
```
Comment (samccann) [inline models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml]:
```suggestion
      description: A list of LLDP interfaces configurations.
```

File: models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description:
+            - Name of the  lldp interface.
```
Comment (samccann) [inline models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml]:
```suggestion
            - Name of the  LLDP interface.
```

File: models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+          type: str
+          required: True
+        enable:
+          default: True
+          description:
+            - to disable lldp on the interface.
```
Comment (samccann) [inline models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml]:
```suggestion
            - Controls disabling of LLDP on the interface C(name).
```

File: models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+                altitude:
+                  description: Altitude in meters.
+                  type: int
+                datum:
+                  description: Coordinate datum type.
+                  type: str
+                  choices:
+                    - WGS84
+                    - NAD83
+                    - MLLW
```
Comment (samccann) [inline models/vyos/vyos_lldp_interfaces/vyos_lldp_interfaces_model.yaml]: These should be choices: ['WGS84', 'NAD83', 'MLLW']

**Pull request merged**